src/metabase/driver/mysql.clj

```diff
@@ -608,13 +608,14 @@
 (defmethod driver/upload-type->database-type :mysql
   [_driver upload-type]
   (case upload-type
-    ::upload/varchar_255 "VARCHAR(255)"
-    ::upload/text        "TEXT"
-    ::upload/int         "BIGINT"
-    ::upload/float       "DOUBLE"
-    ::upload/boolean     "BOOLEAN"
-    ::upload/date        "DATE"
-    ::upload/datetime    "TIMESTAMP"))
+    ::upload/varchar_255 [:varchar 255]
```

Comment:
I think having to mix types like this is a smell that the return type of this function should be a string, not a keyword or vector of things that can be stringified by HoneySQL. If there's a reason why it has to be a keyword, it would be simpler to use `(keyword "VARCHAR 255")` like we do for H2 [here](https://github.com/metabase/metabase/pull/34173/files#diff-c4bd5c99a5e021fc7f1ecf5305075b8195b50845e23343846284a821fb0e4a3cR580).